Update readme with integrations link & a note about existing users (#5)
nimbella/netlify-plugin-nimbella: README.md

   ```sh
   $ netlify addons:create nimbella
   ```
   The addon will create a namespace where your resources reside. You can claim the namespace by running `netlify addons:auth nimbella`.

2. **Add Nimbella Build Plugin to Your Netlify Site**

   Append the below to your `netlify.toml`.

   ```toml
   [[plugins]]
   package = "netlify-plugin-nimbella"
   ```

### Existing Nimbella User

Existing users need to create a Netlify Build environment variable so the plugin can deploy the resources under their account. Follow the steps below to use your existing account with the plugin.

1. **Create Netlify Build Environment Variable**

>    First, obtain Nimbella login token by running `nim auth export --non-expiring` if you've [`nim`](https://nimbella.io/downloads/nim/nim.html) installed or by visiting [`nimbella.com/login`](https://nimbella.com/login).

Reviewer: ```suggestion
- Make sure you have the [Nimbella CLI `nim`](https://nimbella.io/downloads/nim/nim.html) installed. You can do that from the CLI web page or by visiting [`nimbella.com/login`](https://nimbella.com/login) to login.
- Generate a token using the CLI by running the following command:
   ```bash
   nim auth export --non-expiring
   ```
```

Reviewer: the last triple ticks aren't showing up correctly in the suggestion.

>    Then, go to [`https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment`](https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment) and create an environment variable named `NIMBELLA_LOGIN_TOKEN` with your the token you just obtained.

Reviewer: ```suggestion
   Next, visit [`https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment`](https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment) to create an environment variable named `NIMBELLA_LOGIN_TOKEN` with the token you just obtained.
```
